prometheus/prometheus.yml

```diff
@@ -0,0 +1,15 @@
+scrape_configs:
+  - job_name: "prometheus-prosody"
+    scrape_interval: 5s
+    static_configs:
+      - targets: ["prosody:5280"]
+      
+  - job_name: "prometheus-jvb"
+    scrape_interval: 5s
+    static_configs:
+      - targets: ["jvb:8080"]
+
+  - job_name: "prometheus-jicofo"
```

Comment:
Is there a reason to have separate jobs for each target?

Reply:
This makes separation of concern and better individual service configuration. Suppose, we want to add authorization/authentication for one of the components or change the `scrape_interval` then it would be easier to maintain. 

We can have a single job named `prometheus` and have all the targets listed in there. Then we are required to create a common config with all the features listed for each and every service.

---

Whatever way we choose, it should be best for future use and should have ease of modification.